src/raytracer/multipath.m

```diff
@@ -193,7 +195,7 @@
             dRay(12) = mod(atan2d(doa(2),doa(1)), 360);
             % Aoa elevation
             dRay(13) = acosd(doa(3) / norm(doa));
-            dRay(18) = orderOfReflection*pi;
+            dRay(18) = orderOfReflection*pi+mod(distance/wavelength*2*pi,2*pi);
```

Comment:
why is the mod only on the second part? i would mod the entire expression or nothing

Reply:
i was distracted by the mod, but similar considerations can be done here as in the LOS code